ubuntu/debian/gz-transport12-cli.install

```diff
@@ -0,0 +1,4 @@
+usr/libexec/gz/transport[0-9][0-9]/*
```

Comment:
The files under `ubuntu/debian` are unversioned, this should be `ubuntu/debian/gz-transport-cli.install`

Reply:
Moved in [82cd9b9](https://github.com/gazebo-release/gz-transport12-release/pull/16/commits/82cd9b935e9b6e6a4ffd2a8c4fc5703cadaac477)

Reply:
Moved in [82cd9b9](https://github.com/gazebo-release/gz-transport12-release/pull/16/commits/82cd9b935e9b6e6a4ffd2a8c4fc5703cadaac477)